x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/response/schemas.ts

```diff
@@ -24,6 +24,7 @@ export const file_name = t.string;
  * become the actual ESFilter as a type.
  */
 export const filters = t.array(t.unknown); // Filters are not easily type-able yet
+export const actions = t.array(t.unknown);
```

Comment:
Is this unkown for the same reason as filters? If so, maybe we can add the same comment?